README.md

```diff
@@ -2,44 +2,75 @@
 
 # Home Office Forms Example Form
 
-In order to provide a starting point for people using the [home office forms toolkit](https://github.com/UKHomeOffice/hof) this app aims to give a simple example of how to use the module. We encourage users to clone this repository in order to provide a starting point for their own forms.
+This is an example of a service built with [Home Office Forms](https://github.com/UKHomeOffice/hof), A.K.A HOF. It intends to highlight many of the features available from HOF, but is not exhaustive.
+
+## Items of Note
+
+This is not the only way to configure your HOF service. This service uses a typical Node.JS, Express architecture, with an entry point named `app.js`, which is where the majority of the set up occurs.
+
+The specifics of a form are configured in `apps/`, this is where views, controllers, translations and the steps and fields are defined for each form journey (spoiler: there is only one at the moment).
+
+For more info see the [HOF documentation](https://github.com/UKHomeOffice/hof/blob/master/documentation/index.md) or [join the discussion](https://ukgovernmentdigital.slack.com/messages/hof/)
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
 ### Prerequisities
 
-What things you need to install the software and how to install them
-- [NodeJS](https://nodejs.org/en/)
-- npm (bundled with node)
+- [NodeJS](https://nodejs.org/en/) - version 4 and up.
+- npm version 3 (bundled with node 5 and up)
 - [Redis server](http://redis.io/topics/quickstart) running on the default port
+- A healthy understanding of Node.JS
 
 ### Installing
 
-```bash
-$ cd hof-example-form
-$ npm install
-$ npm run dev
-```
+1. Clone this repository: `git clone git@github.com:UKHomeOffice/hof-example-form`
+
+2. Delete the commit history: `rm -rf .git`
```

Comment:
This is a bit mean. The user probably doesn't care about the commit history at this point and it's really their lookout anyway.

If we don't want people to inherit the commit history for some reason then it would probably make more sense to tell them to download an archive rather than cloning.